packages/yourloops/pages/profile/auth-form.tsx

```diff
@@ -80,17 +84,24 @@ function AuthenticationForm(props: AuthenticationFormProps): JSX.Element {
         value={currentPassword}
         error={errors.currentPassword}
         helperText={t("no-password")}
-        setState={setCurrentPassword}
+        onChange={setCurrentPassword}
       />
       <Password
         id="profile-textfield-password"
         autoComplete="new-password"
         variant="standard"
         label={t("new-password")}
         value={password}
-        error={errors.password}
-        helperText={t("password-too-weak")}
-        setState={setPassword}
+        error={errors.password && password.length > 0}
+        checkStrength
+        helperText={
```

Comment:
Need to set FormHelperTextProps too

Reply:
after refactoring the helperText condition into paswword component, error disappear